apps/documentation/src/routes/documentation/index.mdx

```diff
@@ -12,39 +12,43 @@ import Breadcrumbs from '../../components/breadcrumbs'
 
 # Tuono
 
-Tuono is a full-stack framework for building React applications using Rust as the
-backend.
+Tuono is a full-stack web framework for building React applications using Rust as the
+backend with a strong focus on usability and performance.
 
 Tuono (Italian word for "thunder", pronounced /2 Oh No/). Why Tuono? Just a badass name.
 
+If you have experience with [Next.js](https://nextjs.org/) you will feel home here.
+
+## Features
+
+- 🟦 Native Typescript
+- 🌐 Next.js like Routing
+- 🍭 CSS/SCSS modules
+- 🧬 Server Side Rendering
+- 🔥 Hot Module Reload
+
 ## Introduction
 
-NodeJs/Deno/Bun are the only runtimes that allow a React app to be full-stack right? (no)
+Tuono addresses the following two limits:
 
-Tuono is a full-stack React framework with the server side written in Rust.
-Because of this, Tuono is extremely fast and, the requests are handled by a multithreaded Rust server.
+- Enable React server-side rendering projects without using JS runtimes (Node, Deno)
+- Ease the web development using a typically hard language like Rust.
```

Comment:
**[LanguageTool]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>The official spelling of this programming framework is “Node.js”. (NODE_JS[3])
Suggestions: `Node.js`
Rule: https://community.languagetool.org/rule/show/NODE_JS?lang=en-US&subId=3
Category: COMPOUNDING
<!-- __reviewdog__:ChA4MmViMjhlNTBlNTFmMGEzEgxMYW5ndWFnZVRvb2w= -->